### ISSUE TITLE
WINDOWS 10 : ESM Loader absolute path error

### ISSUE THREAD

**Issue by TheRakeshPurohit** [yuanqing/create-figma-plugin#77]
info Building...
error esbuild error
    Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'

**Comment by jamesstoneco** [comment]
Usually this can be fixed by changing the package.json scripts use of `'` and replacing it with an escaped `"` aka `\"`

For example line 10:

```
    "clean": "rimraf '*.log'",
```

becomes

```
    "clean": "rimraf \"*.log\"",
```

It could be more extensive than this, but this is a good starting point.

**Comment by m3t4lch7** [comment]
Our team is also blocked by this, could someone take a look? Thanks

**Comment by fwextensions** [comment]
I just ran into this as well.  All I did was add a `build-figma-plugin.main.js` file to the top-level directory.  It contained just this code:

```
module.exports = function (buildOptions) {
	console.log(buildOptions);
	return buildOptions;
}
```

I assume the CLI is passing a raw Windows path to this file or its output to esbuild, hence the `Received protocol 'c:'` message.  It's probably something like `C:\MyPlugin\build-figma-plugin.main.js`.

**Comment by fwextensions** [comment]
I think the `build-figma-plugin.main.js` file is imported [here](https://github.com/yuanqing/create-figma-plugin/blob/8b63d7bb9a12351d4d3a43ab7b9085e62f7e9f43/packages/build/src/utilities/build-bundles-async/build-bundles-async.ts#L44).  It's resolving an absolute path to the build override file and then dynamically importing it.  

From [this node bug](https://github.com/nodejs/node/issues/31710) (which is "by design" but continues to trip people up), it looks like absolute paths need to be passed in via `file://` URL, but strings starting with `/` are considered relative to the URL base, which on Mac is just the root of the file system.  But `path.resolve()` returns a `C:\` absolute path on Windows, which then causes `import()` to fail.  

I think the solution is just to use a relative path, if the CWD is set correctly, like `await import('./' + esbuildConfigFilePath)`.  Or maybe `new URL('./' + esbuildConfigFilePath, import.meta.url)` to get an absolute `file://` URL.

**Comment by farhadi-erfan** [comment]
Has anyone been able to fix this issue?

**Comment by fwextensions** [comment]
It looks like this was maybe fixed for the main.js file, but it still happens for the `build-figma-plugin.manifest.js` file.

**Comment by TheUltDev** [comment]
Fixed in #211

**Comment by yuanqing** [comment]
This should be fixed as of `3.2.0`
